pages/library/_viewId.vue

```diff
@@ -157,6 +198,127 @@ export default Vue.extend({
     }
   },
   methods: {
+    async filterMedia() {
+      try {
+        const collectionInfo = await this.$api.items.getItems({
+          uId: this.$auth.user.Id,
+          userId: this.$auth.user.Id,
+          ids: this.$route.params.viewId
+        });
+
+        if (
+          collectionInfo.data.Items &&
+          (collectionInfo.data.Items[0].Type === 'CollectionFolder' ||
+            collectionInfo.data.Items[0].Type === 'Folder')
+        ) {
+          const options: any = {
```

Comment:
```suggestion
          const options: ItemsApiGetItemsRequest = {
```

Avoid using `any` when types are available.

Reply:
When I chang eit to `ItemsApiGetItemsRequest` I get errors regarding `Cannot assign to 'filters' because it is a read-only property`.

Reply:
Would it be okay if I changed some of the `ItemsApiGetItemsRequest` properties from readonly to not readonly?